src/icrar/leap-accelerate/tests/math/vector_tests.cc

```diff
@@ -53,10 +53,12 @@ class vector_tests : public testing::Test
     {
         if(useCuda)
         {
+#ifdef CUDA_ENABLED
```

Comment:
Same comment regarding rutime v/s compile time check, same for the rest of the file.

Reply:
Disregard my previous comment, this seem like a legitimate need for the macro now.